cookbook/controller/service.rst

```diff
@@ -232,6 +234,28 @@ inject *only* the exact service(s) that you need directly into the controller.
    are valid, exactly how you want to organize your reusable code is up to
    you.
 
+Base Controller Methods and their service replacements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This table provides an overview how to do the convenience methods of the base
+controller yourself.
+
+=========================== ================================== ==========================================================================================================================================
```

Comment:
In the rest of the documentation, when we use this simple table syntax, we add two white spaces between columns to improve readability.